core/src/test/scala/unit/kafka/consumer/ZookeeperConsumerConnectorTest.scala

```diff
@@ -322,10 +322,10 @@ class ZookeeperConsumerConnectorTest extends KafkaServerTestHarness with Logging
 
   @Test
   def testLeaderSelectionForPartition() {
-    val zkClient = ZkUtils.createZkClient(zkConnect, 6000, 30000)
+    val zkUtils = ZkUtils.apply(zkConnect, 6000, 30000, false)
```

Comment:
With apply, you can just instantiate using ZkUtils(zkConnect, 6000, 30000, false).